### DISCUSSION TITLE
osu! lazer freezing DE or X server (uncertain)

### DISCUSSION THREAD

**Discussion by ludgie** [ppy/osu#17771] [Q&A] [answered]
Hello, I've downloaded today osu!lazer 2022.409.0 build, and when I ran it, my DE went froze, and I wasn't able to do anything (even restarting via menus or anything), although the initial welcome sound was playing (welcome to OSU!...).
I was able to access tty, but I ended up restarting, here are the logs:
[runtime.log](https://github.com/ppy/osu/files/8464635/runtime.log)
[performance.log](https://github.com/ppy/osu/files/8464637/performance.log)
[network.log](https://github.com/ppy/osu/files/8464640/network.log)
[database.log](https://github.com/ppy/osu/files/8464641/database.log)

Neofetch:
![image](https://user-images.githubusercontent.com/77110723/162754520-94cbffaa-49b5-4dab-a894-7069f1edc1f3.png)

**Comment by peppy** [comment, marked as answer]
Please look for technical support for your linux distribution or graphics board elsewhere. This is not an osu! issues (osu! cannot freeze your system) and we can't help with general linux tech support.
